services/servers/tg-bots/publisher/utils/addChannel.ts

```diff
@@ -36,21 +37,21 @@ export const addChannel = async (ctx, text) => {
   try {
     chat = await ctx.telegram.getChat(`@${channel}`)
   } catch (error) {
-    await ctx.reply(i18n['ru'].message.checkChannelName)
+    await ctx.reply(i18n['ru'].message.checkChannelName())
     return
   }
   if (chat.type === 'private') {
-    await ctx.reply(i18n['ru'].message.addedUserInsteadOfChannel)
+    await ctx.reply(i18n['ru'].message.addedUserInsteadOfChannel())
 
     logUserAction(ctx.from, {
       ...logEntity,
-      error: `Adding private channel`,
+      error: `Adding a private channel`,
       channel,
     })
     return
   }
-  let isOurUserAnAdmin: boolean
-  let isOurBotAnAdmin: boolean
+  let isOurUserAnAdmin: boolean | undefined
```

Comment:
Why is `undefined` possible?

Reply:
I want to add the `"strict": true` rule into ts config. So... Let variables are undefined until we define them.